nixos/modules/services/security/endlessh-go.nix

```
@@ -0,0 +1,107 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+
+with lib;
+
+let
+  cfg = config.services.endlessh-go;
+in
+{
+  options.services.endlessh-go = {
+    enable = mkEnableOption "endlessh-go service.";
```

```suggestion
    enable = mkEnableOption "endlessh-go service";
```
the `.` is added too